news/admin.py

```diff
@@ -0,0 +1,100 @@
+from django.conf import settings
+from django.contrib import admin
+
+from .models import Article, ArticleTag, Author
+
+
+@admin.register(ArticleTag)
+class ArticleTagAdmin(admin.ModelAdmin):
+    """
+    Admin config for ArticleTag model
+    """
+
+    list_display = getattr(settings, "ARTICLETAG_ADMIN_LIST_DISPLAY", ["tag"])
```

Comment:
Not a huge deal but it might be worth writing a `get_setting(name: str, default)` function so there's slightly less repetition going on.

Reply:
Have added as a potential todo